barretenberg/cpp/src/barretenberg/protogalaxy/protogalaxy_prover_internal.hpp

```diff
@@ -66,11 +66,12 @@ template <class DeciderProvingKeys_> class ProtogalaxyProverInternal {
      * representing the sum f_0(ω) + α_j*g(ω) where f_0 represents the full honk evaluation at row 0, g(ω) is the
      * linearly dependent subrelation and α_j is its corresponding batching challenge.
      */
-    static std::vector<FF> compute_row_evaluations(const ProverPolynomials& polynomials,
-                                                   const RelationSeparator& alpha,
-                                                   const RelationParameters<FF>& relation_parameters)
+    static std::vector<FF> compute_row_evaluations(BB_UNUSED const ProverPolynomials& polynomials,
```

Comment:
what's going on here with these unused tags? doesn't seem to apply..

Reply:
Debugging leftover - will clean up